spec/locales-currencies-tz.html

```diff
@@ -164,59 +164,229 @@ <h1>
+            1. Set _primary_ to the value in the second column of the row where the first column is _identifier_.
+          1. Else,
+            1. NOTE: The algorithm steps below are intended to mimic the behaviour of *icu::TimeZone::getIanaID()* in the International Components for Unicode (ICU) <a href="https://icu.unicode.org/">https://icu.unicode.org/</a>.
+               The steps in this section are provided for testing conformance of ICU-based ECMAScript implementations, and to ensure compatibility between ICU-based and non-ICU-based implementations.
+            1. If _identifier_ is present in the *TZ* column of *zone.tab* of the IANA Time Zone Database, then
+              1. Set _primary_ to _identifier_.
+            1. Else if _identifier_ is a Link name in the IANA Time Zone Database, then
+              1. Let _zone_ be the Zone name that _identifier_ resolves to, according to the rules for resolving Link names in the IANA Time Zone Database.
+              1. If _zone_ starts with *"Etc/"*, then
+                1. Set _primary_ to _zone_.
+              1. Else,
+                1. Let _identifierCountryCode_ be the ISO 3166-2 country code <a href="https://www.iso.org/iso-3166-country-codes.html">https://www.iso.org/iso-3166-country-codes.html</a> whose territory contains the geographical area corresponding to _identifier_.
+                1. Let _zoneCountryCode_ be the ISO 3166-2 country code whose territory contains the geographical area corresponding to _zone_.
+                1. If _identifierCountryCode_ is _zoneCountryCode_, then
+                  1. Set _primary_ to _zone_.
+                1. Else,
+                  1. Let _countryCodeLine_ be the line in *zone.tab* of the IANA Time Zone Database where the *country-code* column is _identifierCountryCode_.
+                  1. Set _primary_ to the contents of the *TZ* column of _countryCodeLine_.
+            1. If _primary_ is one of *"Etc/UTC"*, *"Etc/GMT"*, or *"GMT"*, set _primary_ to *"UTC"*.
+            1. If _primary_ is a replacement time zone identifier, and it has been less than two years since the release of the IANA Time Zone Database that added _primary_ as a new Zone name, then
+              1. Let _renamedIdentifier_ be the renamed time zone identifier that _primary_ replaced.
+              1. Set _primary_ to _renamedIdentifier_.
+              1. NOTE: This two year waiting period does not need to be exact, and is not required to be applied dynamically, especially in implementations that do not update time zone data between releases.
+                Instead, implementations may make the replacement time zone identifier primary as part of their normal release process for updating time zone data, and to release the change as close as practical to the end of the two year waiting period.
```

Comment:
```suggestion
            1. If _primary_ is a replacement time zone identifier, and the corresponding rename waiting period since the release of the IANA Time Zone Database that added _primary_ as a new Zone name has not concluded, then
              1. Let _renamedIdentifier_ be the renamed time zone identifier that _primary_ replaced.
              1. Set _primary_ to _renamedIdentifier_.
```